File: Homeworks/hw1_java/src/main/java/com/pascal/homeworks/SecondActivity.java
package com.pascal.homeworks;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class SecondActivity extends AppCompatActivity {
    private Button btnToFirstAct;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        btnToFirstAct = findViewById(R.id.back_btn);
        btnToFirstAct.setOnClickListener((View v) -> {
                    Intent intent = new Intent(this, MainActivity.class);
                    startActivity(intent);
        });
    }

}

File: Homeworks/hw1_kotlin/src/main/java/com/pascal/hw1_kotlin/SecondActivity.kt
package com.pascal.hw1_kotlin

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class SecondActivity : AppCompatActivity() {
    private lateinit var btnToSecAct: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        btnToSecAct = findViewById(R.id.back_btn)
        btnToSecAct.setOnClickListener { view: View ->
            val intent = Intent(this, MainActivity::class.java)
            startActivity(intent)
            finish()
        }
    }


}